test/unit/deprecated.js

```diff
@@ -165,6 +165,67 @@ QUnit.test( "jQuery.nodeName", function( assert ) {
 	);
 } );
 
+QUnit.test( "core", function( assert ) {
```

Comment:
`basic.js` is supposed to cover very basic tests from all modules so that we can run it (in the future) in some restricted environments like Node.js with jsdom. All its assertions should already be covered by other modules so it doesn't make sense to move it to `deprecated.js`.

Even its name is misleading here as only `jQuery.type` assertions are left.